src/index.js

```diff
@@ -118,29 +100,41 @@ window.addEventListener("load", () => {
     }
     breakMin.innerHTML = breakTime;  
   };
-
-  const getNotificationMessage = isBreak => {
-    return isBreak ? 'Take a break!' : 'Keep working!';
-  };
   
   const showingAlert = () => {
-    chrome.notifications.create(`pomodoro_alert_${Math.random()}`, {
-      type: 'basic',
-      iconUrl: 'assets/images/tomato.png',
-      title: getNotificationMessage(isBreak),
-      message: 'Timer update!',
-      priority: 2
-    })
+    if (typeof chrome?.notifications?.create === "function") {
+      chrome.notifications.create(`pomodoro_alert_${Math.random()}`, {
+        type: 'basic',
+        iconUrl: 'assets/images/tomato.png',
+        title: getNotificationMessage(isBreak),
+        message: 'Timer update!',
+        priority: 2
+      })
+    } else {
+      alert(getNotificationMessage(isBreak))
+    }
   };
   
-  // describe('#getNotificationMessage', () => {
-  //   it('returns "Take a break!" when it is break time', () => {
-  //     expect(getNotificationMessage(true)).toBe('Take a break!');
-  //   });
-  //   it('returns "Keep working!" when it is work time', () => {
-  //     expect(getNotificationMessage(false)).toBe('Keep working!');
-  //   })
-  // })
-  
   timerEvents();
 });
+
+export const calculate = (a, b) => {
+  return a + b;
+};
+
+export const getNotificationMessage = (isBreak) => {
+  return isBreak ? 'Take a break!' : 'Keep working!';
+}; 
+
+export const buttonDisplay = (isPaused, countdown) => {
+  if (isPaused && countdown === 0) { // beginning/ first iteration
+    buttonUpdate("START");
```

Comment:
return the text from here, don't make a function call here.

Reply:
and instead of calling `buttonDisplay()` everywhere, you need to do `buttonUpdate(butttonDisplay(..params))`

Reply:
😯😯😯😯😯
Thank you!